.kiro/specs/scrollspy-toc/design.md

```
@@ -0,0 +1,450 @@
+| TOCAvatar | UI/Presentation | アクティブ項目のアバター表示 | 6.1-6.4 | - | State |
+
+### Hooks
+
+#### useScrollSpy
+
+| Field | Detail |
+|-------|--------|
+| Intent | Intersection Observerを使用してアクティブな見出しを検出 |
+| Requirements | 2.1, 2.2, 2.3, 2.4 |
+
+**Responsibilities & Constraints**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)